packages/x-date-pickers-pro/src/MobileDateRangePicker/MobileDateRangePicker.test.tsx

```diff
@@ -24,7 +24,7 @@ const WrappedMobileDateRangePicker = withPickerControls(MobileDateRangePicker)({
 });
 
 describe('<MobileDateRangePicker />', () => {
-  const { render } = createPickerRenderer();
+  const { render } = createPickerRenderer({ clock: 'fake' });
```

Comment:
How do you decide whether to use real or fake clock?
I usually try not to use fake clock unless I have to, but maybe I'm missing something?

Reply:
After selecting the dates there's a transition. If I run it with the real timer, this test fails with `An update to Transition inside a test was not wrapped in act(...).`. The idea of using the fake timer is to not trigger the transition and make it pass. I don't  understand quite well why it fails, because `userEvent.mousePress` uses `fireEvent.click` which already wraps in `act`. I also avoid fake timers but every time that there's a situation of a transition running or some update based on timeout I need to use them to not trigger the warning above.